build/yamls/antrea-windows-with-ovs.yml

```diff
@@ -4,39 +4,12 @@ data:
     $ErrorActionPreference = "Stop"
     $mountPath = $env:CONTAINER_SANDBOX_MOUNT_POINT
     $mountPath = ($mountPath.Replace('\', '/')).TrimEnd('/')
-    $OVSDriverDir = "$mountPath\openvswitch\driver"
-
-    # Check if OVSExt driver is already installed
-    $driverStatus = netcfg -q ovsext
-    if ($driverStatus -like '*not installed*') {
-      # Install OVS Driver
-      $result = netcfg -l $OVSDriverDir/ovsext.inf -c s -i OVSExt
-      if ($result -like '*failed*') {
-        Write-Host "Failed to install OVSExt driver: $result"
-        exit 1
-      }
-      Write-Host "OVSExt driver has been installed"
-    }
-
-    # Check if the VC redistributable is already installed.
-    $OVSRedistDir="$mountPath\openvswitch\redist"
-    if (Test-Path $OVSRedistDir) {
-      $dllFound = $false
-      $paths = $env:PATH -split ';'
-      foreach ($path in $paths) {
-        $dllFiles = Get-ChildItem -Path $path -Filter "vcruntime*.dll" -File -ErrorAction SilentlyContinue
-        if ($dllFiles.Count -gt 0) {
-          $dllFound = $true
-          break
-        }
-      }
-
-      # vcruntime dlls are not installed on the host, then install the binaries.
-      if (-not $dllFound) {
-        Get-ChildItem $OVSRedistDir -Filter *.exe | ForEach-Object {
-          Start-Process -FilePath $_.FullName -Args '/install /passive /norestart' -Verb RunAs -Wait
-        }
-      }
+    $OVSInstallScript = "$mountPath\k\antrea\Install-OVS.ps1"
+    if (Test-Path $OVSInstallScript) {
+      & $OVSInstallScript -InstallUserspace $false -LocalFile $mountPath\openvswitch
```

Comment:
In case user want ot bring their own signed driver, User must be able to know if driver is already installed or not or failed to install , it might not be preferable to check log for failure later.
Instead  Start-Job can also be used to schedule the script  in background asynchronously, while capturing the status of the background script Install-OVS.ps1 installation using wait-job as and when it returns success or error  ! and then just validate driver installation success or failure here as it may allow user to know if driver is installed ? and then they might want to uninstall it and use their own signed driver.

Reply:
Got it , I can try with command "Start-Job" to run the script.